src/Mod/Sketcher/Gui/ViewProviderSketch.cpp

```diff
@@ -1738,6 +1461,293 @@ bool ViewProviderSketch::mouseMove(const SbVec2s& cursorPos, Gui::View3DInventor
     return false;
 }
 
+void ViewProviderSketch::initDragging(int geoId, Sketcher::PointPos pos, Gui::View3DInventorViewer* viewer)
+{
+    if (geoId < 0) {
+        return; // don't drag externals
+    }
+
+    drag.reset();
+    Mode = STATUS_SKETCH_Drag;
+    drag.Dragged.push_back({ geoId, pos });
+
+    // Adding selected geos that should be dragged as well.
+    for (auto& geoIdi : selection.SelCurvSet) {
+        if (geoIdi < 0) {
+            continue; //skip externals
+        }
```

Comment:
This could really use `isExternal(int geoId)` helper, just to make this readable without comments

Reply:
Not opposed to the idea, but note that the above include the axes. `isExternal` is different than `isNotInternal`.

Reply:
Ah yes you are correct, we don't want the axes so isExternal is not goog here